src/tokens.ts

```diff
@@ -130,6 +130,14 @@ export interface IDebugger extends IDisposable {
    * Removes all the breakpoints from the current notebook or console
    */
   clearBreakpoints(): Promise<void>;
+
+  /**
+   * Retrieve the content of a source file.
+   * @param source The source object containing the path to the file.
+   */
+  getSource(
+    source: DebugProtocol.Source
+  ): Promise<DebugProtocol.SourceResponse>;
```

Comment:
This could instead return a different interface than `DebugProtocol.SourceResponse`, that would only contain `content` and `mimeType`.

Reply:
I agree with this.

Reply:
This can be done in another PR, though